src/main/java/banking/database/Operations.java

```diff
@@ -0,0 +1,149 @@
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return false;
+    }
+
+    public int login(String cardNumber, String PIN) {
+        ResultSet res;
+        try (PreparedStatement preparedStatement = DriverManager.getConnection(url).prepareStatement(SELECT_ID)) {
+            preparedStatement.setString(1, cardNumber);
+            preparedStatement.setString(2, PIN);
+            res = preparedStatement.executeQuery();
+            if (res.next()) {
+                return res.getInt("id");
+            } else {
+                return -1;
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return -1;
+    }
+
+    public int getBalance(int id) {
+        int balance = -1;
+        try (PreparedStatement preparedStatement = DriverManager.getConnection(url).prepareStatement(GET_CARD)) {
+            preparedStatement.setInt(1, id);
+            ResultSet res = preparedStatement.executeQuery();
+            assert res != null;
+            if (res.next()) {
+                balance = res.getInt("balance");
+                return balance;
+            }
+        } catch (SQLException throwables) {
+            throwables.printStackTrace();
+        }
+        return balance;
+    }
+
+    public String getNumber(int id) {
+        String number = "";
+        try (PreparedStatement preparedStatement = DriverManager.getConnection(url).prepareStatement(GET_CARD)) {
+            preparedStatement.setInt(1, id);
+            ResultSet res = preparedStatement.executeQuery();
+            assert res != null;
+            if (res.next()) {
+                number = res.getString("number");
+                return number;
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return number;
+    }
+
+    public void addBalance(int id, int amount) {
+        try (PreparedStatement preparedStatement = DriverManager.getConnection(url).prepareStatement(UPD_BALANCE)) {
+            preparedStatement.setInt(1, amount);
+            preparedStatement.setInt(2, id);
+            preparedStatement.executeUpdate();
+        } catch (SQLException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    public void sendMoney(int id, String card, int amountSend) {
+        try (PreparedStatement preparedStatement = DriverManager.getConnection(url).prepareStatement(GET_MONEY);
+             PreparedStatement preparedStatement1 = DriverManager.getConnection(url).prepareStatement(SEND_MONEY)) {
+            preparedStatement.setInt(1, amountSend);
+            preparedStatement.setString(2, card);
+            preparedStatement1.setInt(1, amountSend);
+            preparedStatement1.setInt(2, id);
+            preparedStatement.executeUpdate();
+            preparedStatement1.executeUpdate();
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+        }
+    }
```

Comment:
Can refactor this method and split to smaller pieces?

Reply:
fixed